### PR TITLE
Replaced usage of Object.assign with the MDN polyfill

### PR DESCRIPTION
**Pull request by scott113341**
Addresses #48

### CODE REVIEW

**Comment by scott113341** [comment]
Coveralls bot not happy 😭

**Comment by rtsao** [comment]
The coverage won't decrease too 😉

**Comment by scott113341** [comment]
How's that?

File: lib/merge-properties.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+ * Does not modify source objects
+ * @param {...object} - the objects to be merged
+ * @returns {object}  - the new object
+ */
+module.exports = function() {
+  var target = {};
+
+  for (var index = 0; index < arguments.length; index++) {
+    var source = arguments[index];
+    for (var key in source) {
```
Comment (neurosnap) [inline lib/merge-properties.js]: `Object.keys(source).forEach()` is preferable to `for (var key in source)`

https://github.com/airbnb/javascript/issues/851

Comment (scott113341) [inline lib/merge-properties.js]: @neurosnap, from what I understand, `Object.keys` performance is not as good as `for...in`.  I'm working on some [benchmarking](https://github.com/scott113341/csjs-benchmarks) for csjs, but I believe that 5a1434c is as fast as this function can be.  Please correct me if I'm wrong!

Comment (neurosnap) [inline lib/merge-properties.js]: `Object.keys` is more of a stylistic consideration, so you might be right.

**Pull request merged**